Userland/Libraries/LibJS/Console.cpp

```diff
@@ -76,10 +76,14 @@ ThrowCompletionOr<Value> Console::warn()
     return js_undefined();
 }
 
-Value Console::clear()
+// 1.1.2. clear(), https://console.spec.whatwg.org/#clear
+ThrowCompletionOr<Value> Console::clear()
 {
+    // 1. TODO: Empty the appropriate group stack.
+
+    // 2. If possible for the environment, clear the console. (Otherwise, do nothing.)
     if (m_client)
-        return m_client->clear();
+        m_client->clear();
     return js_undefined();
```

Comment:
If `m_client->clear()` is not allowed to fail (it returns void) and we unconditionally return js_undefined() this method could just return `Value` instead of  `ThrowCompletionOr<Value>`.

It is somewhat inconsistent but does clarify that it can't fail.

Reply:
Sure!